camel/models/openai_audio_models.py

```diff
@@ -0,0 +1,224 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import os
+from typing import Any, List, Union
+
+from openai import OpenAI, _legacy_response
+
+from camel.types import AudioModelType, VoiceType
+
+
+class OpenAIAudioModels:
+    r"""Provides access to OpenAI's Text-to-Speech (TTS) and Speech_to_Text
+    (STT) models."""
+
+    def __init__(
+        self,
+    ) -> None:
+        r"""Initialize an instance of OpenAI."""
+        url = os.environ.get('OPENAI_API_BASE_URL')
+        self._client = OpenAI(timeout=120, max_retries=3, base_url=url)
+
+    def text_to_speech(
+        self,
+        input: str,
+        model_type: AudioModelType = AudioModelType.TTS_1,
+        voice: VoiceType = VoiceType.ALLOY,
+        **kwargs: Any,
+    ) -> Union[
+        List[_legacy_response.HttpxBinaryResponseContent],
```

Comment:
It seems that this "legacy" response will be deprecated soon. Do we have any other return type?

Reply:
I didn't find alternative type from OpenAI, when legacy is deprecated I think they should introduce a new type, let's do the update after openai's update~